src/main/java/HW22DB/CustomerCreateException.java

```diff
@@ -0,0 +1,7 @@
+package HW22DB;
+
+public class CustomerCreateException extends Exception{
```

Comment:
Is there an entity called 'customer' in the program? 
Would make sense to rename it correspondingly